tock/tock/settings/dev.py

```diff
@@ -54,6 +54,8 @@ def show_django_debug_toolbar(request):
     'SHOW_TOOLBAR_CALLBACK': 'tock.settings.dev.show_django_debug_toolbar',
 }
 
+USE_X_FORWARDED_HOST = True
```

Comment:
This is required for Django to think that the server it's running on is where Nginx is, rather than where Django runserver is running.

If folks are developing the site on something *other* than Docker though, we may want to put this behind an `if` statement of some sort that only gets executed when on Docker.